h2o-bindings/bin/custom/python/gen_targetencoder.py

```diff
@@ -15,8 +15,22 @@ def transform(self, frame, data_leakage_handling="None", noise=-1, seed=-1):
         :param int seed: a random seed used to generate draws from the uniform distribution for random noise. Defaults to -1.
 
         :example:
-        >>> targetEncoder = TargetEncoder(encoded_columns=te_columns, target_column=responseColumnName, blended_avg=True, inflection_point=10, smoothing=20)
-                            >>> encodedTrain = targetEncoder.transform(frame=trainFrame, data_leakage_handling="None", seed=1234, is_train_or_valid=True)
+        >>> titanic = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/gbm_test/titanic.csv")
+        >>> predictors = ["home.dest", "cabin", "embarked"]
+        >>> response = "survived"
+        >>> titanic["survived"] = titanic["survived"].asfactor()
+        >>> fold_col = "kfold_column"
+        >>> titanic[fold_col] = titanic.kfold_column(n_folds=5, seed=1234)
+        >>> titanic_te = H2OTargetEncoderEstimator(k = 0.7,
+        ...                                        f = 0.3,
+        ...                                        data_leakage_handling = "None",
+        ...                                        blending = True)
+        >>> titanic_te.train(x = predictors,
+        ...                  y = response,
+        ...                  training_frame = titanic)
+        >>> transformed = titanic_te.transform(frame = titanic,
```

Comment:
minor remark: `frame` is a positional/required param, as such, it is not recommended to use it as a keyword param (there's even a new feature in Py 3.8. to prevent this). This works fine, just doesn't improve readability.

minor minoris remark: Py recommendation is to avoid spaces around `=` in function parameters (to distinguish it from `=` as an assignment.
https://www.python.org/dev/peps/pep-0008/#other-recommendations

minimus remark: there's an old typo line 5: `trains()` method doesn't exist.